### PR TITLE
Update region_fullname.txt

### PR DESCRIPTION
**Pull request by FowlerJohn**
One letter "H" was left behind.

### CODE REVIEW

**Comment by CodingWonders** [comment]
This file is not used for anything yet, as it's meant for unattended answer stuff (still not finished). When work begins, all the region contents will be moved to the program resources and these files will be removed.

Also, I'd like to wait until the changes in the preview branches have been merged into the stable branch before merging this one.

Still, thanks for your contribution!

**Comment by FowlerJohn** [comment]
No problem.
Since I'm Hungarian, I noticed.
It is a promising project.
If the localization comes up, I will be happy to contribute.
Hi,
[Fowler](https://forums.mydigitallife.net/members/fowler.355408/).

**Review by CodingWonders — APPROVED** [review]
Good job spotting this issue!

**Pull request merged**
